osbs/build/build_request.py

```diff
@@ -480,7 +480,8 @@ def adjust_for_registry_api_versions(self):
 
         if 'v1' not in versions:
             # Remove v1-only plugins
-            for phase, name in [('postbuild_plugins', 'pulp_push')]:
+            for phase, name in [('postbuild_plugins', 'pulp_push'),
+                                ('exit_plugins', 'pulp_publish')]:
```

Comment:
No, that's not right. The pulp_publish plugin is required regardless of which API versions are used.